docs/specs/markdown.yml

```diff
@@ -17,22 +17,47 @@ outputs:
     { "content": "<p><em>this</em> <em>is</em> an <em><strong>inline</strong></em> <code>markdown</code> <strong>test</strong></p>" }
   build.manifest:
 ---
-# Yaml header
+# Yaml header title in lower-case
 inputs:
   docfx.yml:
   docs/a.md: |
     ---
-    title: me
+    title: title from yaml header
     ---
+    # Title from H1
     hello
 outputs:
   docs/a.json: |
     {
-      "content": "<p>hello</p>",
-      "metadata": { "title": "me" }
+      "content": "<p>hello</p>\n",
+      "titleHtml": "<h1 id=\"title-from-h1\">Title from H1</h1>",
+      "title": "title from yaml header",
+      "metadata": { "title": "title from yaml header" }
     }
   build.manifest:
 ---
+# Yaml header title in upper-case
+inputs:
+  docfx.yml:
+  docs/a.md: |
+    ---
+    Title: Title from yaml header
```

Comment:
if we want to enforce casing here, we should also enforce camel casing during `JToken -> T`:

Deserialize `{ AB: 1 }` into `class { int AB }` should produce `{ AB: 0 }`
Deserialize `{ aB: 1 }` into `class { int AB }` should produce `{ AB: 1 }`

@live1206